Automattic/wp-calypso
# Checkout: Remove Payment Method a/b test

#### Changes proposed in this Pull Request

This PR removes the payment method a/b test documented here: p8Eqe3-RL-P2. The test is over and due to the temporary solution for implementing it as well as the results being flat-ish, I have opted to keep the tabs as is.

#### Testing instructions
* Add a product to our cart and go to checkout. 
* You should only see the tabs and no radio buttons.
